### PR TITLE
try fix last deployed release

### PR DESCRIPTION
**Pull request by jason-liew**
Signed-off-by: Jason Liew <jason_liew@163.com>

### CODE REVIEW

**Comment by jason-liew** [comment]
[#6850 comment](https://github.com/helm/helm/issues/6850#issuecomment-566032294)

**Comment by bacongobbler** [comment]
Hi @jason-liew. Would you mind writing a unit test to cover the case you are trying to fix? That would be much appreciated. Have a look at [storage_test.go](https://github.com/helm/helm/blob/master/pkg/storage/storage_test.go) for ideas. Thanks!

**Review by bacongobbler — CHANGES_REQUESTED** [review]
needs unit tests to confirm whether this fixes the issue proposed

File: pkg/storage/storage.go [hunk 1]
```diff
@@ -122,7 +122,10 @@ func (s *Storage) Deployed(name string) (*rspb.Release, error) {
 		return nil, errors.Errorf("%q has no deployed releases", name)
 	}
 
-	return ls[0], err
+	// We want oldest to newest
+	relutil.SortByRevision(ls)
```
Comment (jason-liew) [inline pkg/storage/storage.go]: usually the order is correct without sort. may be some storage driver need sort.

**Comment by jason-liew** [comment]
Hi, @bacongobbler unit test updated. Thanks.

**Comment by jason-liew** [comment]
some similar pr has been merged

**Pull request closed without merging**